js/viz.js

```diff
@@ -168,6 +170,8 @@ const viz = {
   addListeners() {
     this.addMouseMove();
     this.addWindowResize();
+    // this.addDrag();
+    this.addZoom();
   },
```

Comment:
Working on for the mouse events not to interfere with drag, zoom and pan behaviours.